### PR TITLE
First PR

### PR DESCRIPTION
**Pull request by alejobh**
## Summary

The purpose of this branch is to push the basic project structure and files

## Known isssues/Notes
.gitignore is not correct, the next PR will contain the updated .gitignore and the assets folder will be included

## Trello Card

https://trello.com/c/zTuTeKRp

### CODE REVIEW

File: maquetado/scss/application.scss [hunk 1]
```diff
@@ -0,0 +1,3 @@
+.title {
+    color: red;
+}
```
Comment (henryzarza) [inline maquetado/scss/application.scss]: Remember to add a line end the file

File: maquetado/index.html [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+  <title>HTML & CSS Training</title>
+  <meta name="description" content="training">
+  <meta name="author" content="Wolox">
+  <link href="https://fonts.googleapis.com/css?family=Roboto" rel="stylesheet">
+  <link rel="stylesheet"  type="text/css" href="application.css">
+</head>
+<body>
+  <h1 class="title">Hello World</h1> 
+</body>
+</html>
```
Comment (henryzarza) [inline maquetado/index.html]: Remember to add a line end the file

File: maquetado/application.css [hunk 3]
```diff
@@ -0,0 +1,4 @@
+.title {
+  color: red; }
+
+/*# sourceMappingURL=application.css.map */
```
Comment (henryzarza) [inline maquetado/application.css]: This it's not necessary, please delete this line

File: maquetado/application.css [hunk 4]
```diff
@@ -0,0 +1,4 @@
+.title {
+  color: red; }
```
Comment (henryzarza) [inline maquetado/application.css]: The correct way of doing this is <br />
`.title {`
`' color: red;`
`}`

**Review by henryzarza — CHANGES_REQUESTED** [review]

File: maquetado/application.css (hunk 3 above)
Comment (braiandickson) [reply maquetado/application.css]: This file should be ignered, as all the css files of the proyect.

File: maquetado/index.html [hunk 5]
```diff
@@ -0,0 +1,15 @@
+<!doctype html>
+
```
Comment (braiandickson) [inline maquetado/index.html]: Delete this blank line.

**Review by henryzarza — APPROVED** [review]

**Pull request merged**